src/io.c

```diff
@@ -1102,6 +1150,11 @@ static void PutLine2(TypOutputFile * output, const Char * line, UInt len)
     }
     else {
         SyFputs(line, output->file);
```

Comment:
Actually ... this ignores `len` which looks like a potential bug? Need to investigate

Reply:
No, this is OK, after inspecting the calling code (a comment should perhaps be added explaining this)